src/index.css

```diff
@@ -82,6 +104,7 @@ button {
 .help {
   text-align: center;
   min-width: 100px;
+  box-shadow: 0px 4px 8px #888;

```

Comment:
Should've done this originally, but since this exact box shadow appears several times, maybe put it inside a new `background-shadow` CSS class and add that class to the buttons (for reusability purposes)?